conmon/utils.c

```diff
@@ -0,0 +1,28 @@
+#include "utils.h"
+
+/* Set the log level for this call. log level defaults to warning.
+   parse the string value of level_name to the appropriate log_level_t enum value
+*/
+void set_conmon_logs(char *level_name, char *cid_, gboolean syslog_)
+{
+	// TODO is this safe? strcpy?
```

Comment:
@giuseppe Considering the cid is always passed from main as the global bool, does the utils library need to copy it, or can it just use the same pointer? I worry about the CID being passed from a different variable down the line, and some funky seg fault action happening, but as written that won't happen.